internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -90,16 +90,30 @@ func translateError(err error) error {
 	}
 }
 
+func foreignKeys(gormdb *gormdb, enable bool) {
```

Comment:
comment why?

Reply:
plz elaborate? 
this is the existing functionality to disable foreign keys on some tests. I've just added PG-related stuff, extracting this to a method

Reply:
```suggestion
// db type -> true/false -> statement
var byDB = map[string]map[bool]string{
	"sqlite": {
		true:  "PRAGMA foreign_keys = ON",
		false: "PRAGMA foreign_keys = OFF",
	},
	"postgres": {
		true:  "SET session_replication_role = DEFAULT",
		false: "SET session_replication_role = replica",
	},
}

func foreignKeys(gormdb *gormdb, enable bool) {
	stmt := byDB[gormdb.cfg.Type][enable]
	gormdb.db.Exec(stmt)
}
```

Reply:
changed according to suggestion

Reply:
> plz elaborate? this is the existing functionality to disable foreign keys on some tests. I've just added PG-related stuff, extracting this to a method

if someone asks for a comment, it means that something is not clear in the code :) you need to explain what each pragma does here and why. it's pretty clear on the sqlite front, but not really on the postgres side.

Reply:
added explanations for PG